src/functions/apis/apis.yml

```diff
@@ -1,6 +1,7 @@
 # MIDDLEWARES
-# - ${file(./src/functions/middlewares/authorization/config.yml)} # Uncomment this if you want to implement middleware
+ - ${file(./src/functions/middlewares/access_token_authorization/config.yml)} # Uncomment this if you want to implement middleware
```

Comment:
Please also include the refresh_token authorizer

Reply:
resolved